src/components/alert/alert.stories.mdx

```diff
@@ -65,6 +65,29 @@ Alerts work well even when displayed at full width. Their contents will constrai
   </Story>
 </Canvas>
 
+<!-- TODO: Right now the icon needs to be set to the sad emoji.
+// As of right now, I don't know how to automatically add the icon on it's own.
+// That is a created issue but not a launch blocker.
+ -->
```

Comment:
Why is this an issue? Shouldn't the user be able to use whatever icon they want?